docs/Security/CVE-2023-23397/index.md

```diff
@@ -102,6 +102,8 @@ ScriptUpdateOnly | This optional parameter allows you to only update the script
 SkipVersionCheck | This optional parameter allows you to skip the automatic version check and script update.
 IgnoreCertificateMismatch | This optional parameter lets you ignore TLS certificate mismatch errors.
 Credential | This optional parameter lets you pass admin credentials when running on Exchange Server.
+UseSearchFolders | This parameter causes the script to use deep-traversal search folders, significantly improving performance.
+SearchFolderCleanup | This parameter cleans up any search folders left behind by the asynchronous search feature.
```

Comment:
Should put in here that `SearchFolderCleanUp` requires `UseSearchFolders` to be enabled as well. OR we need to update the script that if `SearchFolderCleanUp` is set, we need to set `UseSearchFolders`

This was an issue that was brought up in the email thread.

Reply:
Agree with @dpaulson45 on this; we should clarify this in documentation.

Reply:
Done. Clarified that the parameter must be used together with `UseSearchFolders`.